addons/web/tooling/_eslintrc.json

```diff
@@ -2,11 +2,11 @@
   "extends": ["eslint:recommended", "plugin:prettier/recommended"],
   "parserOptions": {
     "sourceType": "module",
-    "ecmaVersion": 2019
+    "ecmaVersion": "latest"
   },
   "env": {
     "browser": true,
-    "es2017": true,
+    "es2020": true,
```

Comment:
Ok for now, but if this is now allowed, we should do it directly in master instead instead of keeping it on our branch.

Reply:
definitely. working on putting that commit in master